src/main/java/it/gov/pagopa/payhub/activities/connector/transfer/config/TransferApisHolder.java

```diff
@@ -0,0 +1,53 @@
+package it.gov.pagopa.payhub.activities.connector.transfer.config;
```

Comment:
under connector package we should have one package per application not one package per entity!
You have to use debtpositions connector, using the same ApisHolder